core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -847,7 +847,8 @@ private void initLMAlgoFactoryDecorator() {
 
         for (FlagEncoder encoder : encodingManager.fetchEdgeEncoders()) {
             for (String lmWeightingStr : lmFactoryDecorator.getWeightingsAsStrings()) {
-                Weighting weighting = createWeighting(new HintsMap(lmWeightingStr), encoder, null);
+                // note that we do not consider turn costs during LM preparation?
+                Weighting weighting = createWeighting(new HintsMap(lmWeightingStr), encoder, null, new NoTurnCostProvider());
```

Comment:
Yes, this is intended. In a separate issue we could try if this would have any negative consequences.